Alarmify/Modules/Edit/Components/DaysSelectorView.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  DaysSelectorView.swift
+//  Alarmify
+//
+//  Created by Swastik Patil on 04/06/24.
+//
+
+import SwiftUI
+
+struct WeekDaysView: View {
+    @EnvironmentObject var alarmdata: AlarmData
+    @State var isclicked = false
+    @State var isclicked2 = false
+    var body: some View {
+        
+        HStack {
+            Toggle(isOn: $isclicked2) {
```

Comment:
Use array and ForEach instead; something like: 

```swift
class WeekDay: ObservableObject, Identifable {
	@Published var isSelected = true
	let index: Int
	let label: String
	var id { index }

	// TODO: init
}

let days: [WeekDay] = Array("MTWTFSS").enumerate().map { WeekDay($0, $1) }

// body
ForEach(days) { day in 
	Toogle(isOn: day.isSelected, label: day.label)
	    .frame(...)
}
```

Reply:
**Struct**
```
struct WeekDay: View {
    @State var isClicked: Bool
    var day: String
    var body: some View {
        Toggle(isOn: $isClicked) {
            Text(day)
                .frame(width: 15, height: 15)
        }.toggleStyle(.button)
    }
}
```
**Array**
```
var days = ["M", "T", "W", "T", "F", "S", "S"]
```
**ForEach**
```
ForEach(days, id:\.self) { days in
                WeekDay(isClicked: isclicked, day: days)
            }
```
This is the best I can implement to my best knowledge